src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(value){
+      result += value;
+      return this;
+    },
```

Comment:
Don't forget about code styles
```suggestion
    add(value) {
      result += value;
      
      return this;
    },
```